pkg/didmethod/httpbinding/resolver.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package httpbinding
+
+import (
+	"crypto/tls"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	errors "golang.org/x/xerrors"
+)
+
+var logger = log.New("aries-framework/didmethod/httpbinding")
+
+// HTTPClientOpts holds options for the HTTP transport implementation of DIDResolver
+// it has a required http.Client instance and optional basePath which is appended to the remove DID Resolver URL
+type HTTPClientOpts struct {
+	client   *http.Client
+	basePath string
+}
+
+// HTTPOpt is an outbound HTTP transport option
+type HTTPOpt func(opts *HTTPClientOpts)
+
+// WithHTTPClientInstance option is for creating an Outbound HTTP transport using an http.Client instance
+func WithHTTPClientInstance(client *http.Client) HTTPOpt {
```

Comment:
Let's remove this option for now - we don't anticipate usage of this function right now.

Reply:
👍

Reply:
OK